.golangci.yml

```diff
@@ -15,8 +15,6 @@ linters-settings:
 
 linters:
   enable-all: true
-  disable:
-    - scopelint         # too many false positives
```

Comment:
It's ok? Now we have no many false positives anymore?

Reply:
No in this repo. In fact, it found a real problem: `path := path` is required for `t.Parallel()`.